content/en/security_platform/cspm/getting_started.md

```diff
@@ -19,30 +19,200 @@ Cloud Security Posture Management is not currently available in this site.
+[2]: https://docs.datadoghq.com/integrations/google_cloud_platform/#metric-collection
+[3]: https://app.datadoghq.com/security/configuration
+{{% /tab %}}
+
+{{% tab "Docker" %}}
+
+### Enable CSPM for Docker
+
+1. Navigate to **Security** > **Setup and Configuration**.
+2. Follow the [in-app instructions][1] to activate CSPM for your account.
+3. On the **Setup and Configuration** > **Host and containers** tab, click the **Docker** tile.
+4. Click **Select API key** to choose the API key you want to use with CSPM.
+5. Run the following command to enable CSPM in your Docker environment:
+
+    ```bash
+    DOCKER_CONTENT_TRUST=1 \
+      docker run -d --name dd-agent \
+      --security-opt apparmor:unconfined \
+      --cap-add SYS_ADMIN \
+      --cap-add SYS_RESOURCE \
+      --cap-add SYS_PTRACE \
+      --cap-add NET_ADMIN \
+      --cap-add IPC_LOCK \
+      -v /var/run/docker.sock:/var/run/docker.sock:ro \
+      -v /proc/:/host/proc/:ro \
+      -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+      -v /etc/passwd:/etc/passwd:ro \
+      -v /etc/group:/etc/group:ro \
+      -v /:/host/root:ro \
+      -v /etc/os-release:/host/etc/os-release \
+      -v /sys/kernel/debug:/sys/kernel/debug \
+      -e DD_COMPLIANCE_CONFIG_ENABLED=true \
+      -e DD_RUNTIME_SECURITY_CONFIG_ENABLED=true \
+      -e DD_SYSTEM_PROBE_ENABLED=true \
+      -e HOST_ROOT=/host/root \
+      -e DD_API_KEY=<DATADOG_API_KEY> \
+      -e DD_SITE="datadoghq.com" \
+      datadog/agent:latest
+    ```
+
+**Note**: To disable CSPM, set `DD_COMPLIANCE_CONFIG_ENABLED` to `false`. Your previous findings and homepage will still be available in-app, but you will not incur additional billing costs.
+
+[1]: https://app.datadoghq.com/security/configuration
+
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
+
+### Enable CSPM for Kubernetes
+
+1. If you haven't already, install the [Datadog Agent][1] (version 7.27+).
+2. Navigate to **Security** > **Setup and Configuration**.
+3. Follow the [in-app instructions][2] to activate CSPM for your account.
+4. Add the following to the `datadog` section of the `values.yaml` file:
+    ```yaml
+    # values.yaml file
+    datadog:
+    [...]
+      # Add this to enable Cloud Security Posture Management and Cloud Workload Security
+      securityAgent:
+        runtime:
+          enabled: true
+        compliance:
+          enabled: true
+    ```
+
+5. Restart the Agent.
+
+**Note**: To disable CSPM, set `compliance` > `enabled` to `false`. Your previous findings and homepage will still be available in-app, but you will not incur additional billing costs.
+
+[1]: https://app.datadoghq.com/account/settings#agent/kubernetes
+[2]: https://app.datadoghq.com/security/configuration
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Visualize the first results
+
+CSPM evaluates resources in increments between 15 minutes and four hours (depending on type). New findings from each scan are generated as soon as the scan completes.
```

Comment:
I think this needs more than what I currently have here. Specifically, I need to provide clearer direction as to what we want customers to see/do when viewing their initial findings. It doesn't need to be more than a brief summary as I will be linking to other articles that provide more in-depth instructions.